### ISSUE TITLE
Unable to set warning for Lowest cell voltage.  

### ISSUE THREAD

**Issue by Richrfl** [FrSkyRC/ETHOS-Feedback-Community#3041]
Hi, gents. I recently started using a calculated sensor as Battery Charge, which calculate the charge as a percentage of the Lipo Voltage. For that to have a good reading I had to set, in the telemetry tab, Lipo voltage range from #number of cells time 3.2 to number of cells times 4.2. That works excellent.
On the Logic Switches tab I had already a switch that will trigger a warning if the minimum cell voltage goes below 3.5V. Any here is were the problems becomes apparent. I installed a fully charged battery, and immediately received low cell voltage, giving me a reading of 4.2V. (An special function reads the voltage value when the Logical Switch triggers). When I went to check the LSwitch settings I found the value was 19.2 (my bat is 6 cells). I tried to drop it back to 3.5, but would not go below the minimum value of the lipo range.
In short, at this point, the switch is comparing the cell value with the minimum LiPo range and giving me a wrong warning. 
I tried to do further investigation in the Simulator, and notice in the settings there are added check blocks for individual cells (picture attached) but the same screen in the TX Doesn't show them (picture).
Therefore, at this point I have to choose either I use charge values or cell values. Experience shows that cell values are way safer than charge. 
TX is Tandem X18 FW 1.4.13. FCC
Software Ethos - X18

![screenshot-2023-11-07-59393](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/9392774/80b214a1-8f21-4098-ad33-b37e8e3c87f8)
![SImulator](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/9392774/ceea62f2-b368-4b7f-b4fc-7dd9f4545099)

**Comment by bsongis-frsky** [comment]
I don't understand, you compare "Lowest" (the lowest cell) to 22.2V?

**Comment by Richrfl** [comment]
Myself, I don't compare. The software does. The problem I see is that even
choosing lipo lowest I cannot set the cell value lower than the value set
as lower lipo range (3.2 x #cells) in the telemetry tab required to
calculate Charge. If I set that value to zero, then I can set the lowest
cell value to, say, 3.5 in the logical switch and get the warning as
expected.

**Comment by mawzthefinn** [comment]
If you set a LiPo range in the LiPo sensor config in telemetry, the range is respected when you setup a logic switch that tests against any voltage value for that sensor. Which means you cannot test any of the individual cell voltages.

I tested this myself. I set the LiPo range to 14-16.8v on one of my models with an FVLS ADV installed, then setup a logic switch a<x to test the lowest cell voltage. Could not set a value below 14v

If no range is set on the sensor, then I can set that LS to a valid voltage.


The percentage calculated sensor needs that range set to work

**Comment by Richrfl** [comment]
Gents, I am a bit confused. Is there any additional info you require from me in reference to this issue?
Also, by trying to walk around this issue, I tried to create a "cell voltage" custom sensor, and I also had issues related to the read voltage. Should I report the issue in this thread or open a new issue?

**Comment by mawzthefinn** [comment]
My last response was for @bsongis-frsky showing I could duplicate the behaviour you are reporting

**Comment by Richrfl** [comment]
Ok. Thank you.

**Comment by bsongis-frsky** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/2caae455-9f81-46e5-9f17-0c3b57fad821)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/d130c8a3-7dcf-4244-a5ec-bfcb39ed27b8)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/1e4c61a6-28f5-4f75-83e8-361d91965401)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/072bc744-3dd2-456d-a56b-2f8a6bf54d54)

**Comment by Richrfl** [comment]
Bert, I apologize for my ignorance but what it is LiPo Count? The set up you described is how to "fix" this issue, because if it is, then I have a lot of questions about it.

**Comment by mawzthefinn** [comment]
Lipo Count is one of the other sensor values for the Lipo sensor, it provides the Cell count for the attached battery.

@bsongis-frsky is demonstrating that it's fixed for all values, not just the lowest cell option.

**Comment by bsongis-frsky** [comment]
Exact. And the fix will be included in the next nightly builds (which are running right now). Given the numbers of new radios / flavours, it should be online tomorrow in the middle of the morning (GB / FR time)

**Comment by Richrfl** [comment]
Gents, thank you very much for the clarification and your support. I'll be looking forward to the release of the new version.
